### DISCUSSION TITLE
Should we release 0.72.9?

### DISCUSSION THREAD

**Discussion by fortmarek** [reactwg/react-native-releases#96] [Patches]
# Should we release 0.72.9?

Current Release: [0.72.8](https://github.com/facebook/react-native/releases/tag/v0.72.8)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

#### Local commits to backport to main

1.

**Comment by siddarthkay** [comment]
Can you please get : https://github.com/facebook/metro/pull/1083 in 0.72.9?

**Comment by cipolleschi** [comment]
Can we merge and ship: https://github.com/facebook/react-native/pull/42075

**Comment by lunaleaps** [comment]
https://github.com/facebook/react-native/pull/42105 to pin monorepo deps and remove * deps, same has been done upstream

**Comment by joe-sam** [comment]
Corrects _LogBox_ to display the actual stack trace  libraries instead of internal code on the debug thread for all custom errors during unhandled promises.
 https://github.com/facebook/react-native/pull/42079

**Comment by tido64** [comment]
I  have a fix for 0.72.8 New Arch not building with `-std=c++20`: https://github.com/facebook/react-native/pull/42138

This does not affect 0.73 as it already contains the fix.

**Reply by fortmarek** [reply]
Merged ✅
